kin-devplatform-sdk/src/main/java/kin/devplatform/exception/BlockchainException.java

```diff
@@ -20,6 +20,7 @@
 	public static final int INSUFFICIENT_KIN = 6004;
 	public static final int TRANSACTION_FAILED = 6005;
 	public static final int ACCOUNT_LOADING_FAILED = 6006;
+	public static final int ACCOUNT_CREATION_TIMEOUT = 6007;
```

Comment:
Just for knowledge - why you choose 6000+

Reply:
I didn't, ecosystem did